dashboard/app/models/levels/studio.rb

```diff
@@ -56,6 +56,8 @@ class Studio < Grid
     delay_completion
     floating_score
     goal_override
+    tap_svg_to_run_and_reset
```

Comment:
Yep!  When set, it adds the "Tap or click to play" overlay:

![image](https://cloud.githubusercontent.com/assets/413693/25676656/aef15e82-2ff7-11e7-9a5d-c51fb207c2aa.png)

I didn't expose it via Level Builder because I'm not sure which other apps support this option.